src/jira/api.rs

```diff
@@ -337,4 +363,25 @@ mod tests {
         let versions = parse_pending_versions(&search, "the-field");
         assert_eq!(expected, versions);
     }
+
+    #[test]
+    fn test_comment_issue() {
+        let jira_config = JiraConfig {
+            host: "the-host".into(),
+            username: "the-jira-user".into(),
+            password: "the-jira-pass".into(),
+            progress_states: Some(vec!["progress1".into()]),
+            review_states: Some(vec!["reviewing1".into()]),
+            resolved_states: Some(vec!["resolved1".into(), "resolved2".into()]),
+            fixed_resolutions: Some(vec!["it-is-fixed".into()]),
+            fix_versions_field: Some("the-versions".into()),
+            pending_versions_field: Some("the-pending-versions".into()),
+            restrict_comment_visibility_to_role: None,
+        };
+        let jira_sess = match JiraSession::new(&jira_config) {
```

Comment:
can simplify to: `let jira_sess = JiraSesssion::new(&jira_config).expect("jira session")`